packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -138,73 +152,45 @@ class CupertinoAlertDialog extends StatelessWidget {
   /// Typically this is a list of [CupertinoDialogAction] widgets.
   final List<Widget> actions;
 
-  /// A scroll controller that can be used to control the scrolling of the message
-  /// in the dialog.
+  /// A scroll controller that can be used to control the scrolling of the
+  /// message in the dialog.
   ///
-  /// Defaults to null, and is typically not needed, since most alert messages are short.
+  /// Defaults to null, and is typically not needed, since most alert messages
+  /// are short. Note that another scroll controller, [actionScrollController],
+  /// is needed for the actions section when there are many actions. So, it
+  /// would be nice to disambiguate this scroll controller by changing to
+  /// 'titleScrollController', TODO(ekbiker): change to "titleScrollController",
```

Comment:
please don't put a TODO in the api docs.
Just put them on their own line, prefixed by `//`.

But in this case, you can just remove the TODO. Either we make this change now, or we never make it. There's no benefit to waiting.

Reply:
fixed

Reply:
I assume you aren't going to make this change. Correct?